schemas/yaml/BibliographicReference.yml

```diff
@@ -164,6 +175,7 @@ examples:
   - authors:
       - firstName: 'Daniel W.'
         lastName:  'Hieber'
+    citationKey:     'CatChiti'
```

Comment:
```suggestion
    citationKey:     'Hieber2018'
```

Reply:
This is just a more realistic example of how the citationKey field would be formatted.